src/main/kotlin/nl/info/zac/notification/Resource.kt

```diff
@@ -31,7 +31,8 @@ enum class Resource(private val code: String) {
     ZAAKOBJECT("zaakobject"),
     ZAAKEIGENSCHAP("zaakeigenschap"),
     ZAAKINFORMATIEOBJECT("zaakinformatieobject"),
-    ZAAKTYPE("zaaktype");
+    ZAAKTYPE("zaaktype"),
+    CALLBACK_URL_TEST("some_resource");
```

Comment:
Same as my other comment - let's be consistent with the names
```suggestion
    TEST("test");
```